README.md

```diff
@@ -282,6 +291,29 @@ The command to fine-tune the model with Intel® optimizations enabled is:
 ipexrun --use_logical_core --enable_tcmalloc src/finetune_model.py --model_config configs/config_base.yml --data_path data/abcnews-date-text.csv --save_path saved_models/gpt2-medium-finetuned-intel --intel
 ```
 
+**For intel xpu training**, 'xpu' device is needed to add to 'python/site-packages/transformers'.    
+It can be changed manually as the following:

```

Comment:
```suggestion
**For Intel GPU training**, device 'xpu' must be added to 'python/site-packages/transformers/training_args.py as shown in the below diff at lines'.    
It can be changed manually follows:

```